tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -176,7 +193,11 @@ def test_no_valid_url(self):
         )
 
         resp = self.get_success_response(
-            self.organization.slug, self.project.slug, event.event_id, frame_idx=0
+            self.organization.slug,
+            self.project.slug,
+            event.event_id,
+            frame_idx=0,
+            exception_idx=0,
```

Comment:
@roggenkemper could you add a case where `exception_idx=1` and maker sure it actually finds the right exception?

Reply:
yup!